plugin/trino-phoenix/src/test/java/io/trino/plugin/phoenix/TestPhoenixTypeMapping.java

```diff
@@ -86,7 +86,7 @@ public void destroy()
     @Test
     public void testBasicTypes()
     {
-        DataTypeTest.create()
+        DataTypeTestToSqlDataTypeTestConverter.create()
```

Comment:
@90lantran Not sure if you were able to notice this already or not. Using `DataTypeTestToSqlDataTypeTestConverter.create` will throw a RuntimeException whose message includes the equivalent Java code that uses SqlDataTypeTest. You can copy that and replace here.

The generated code might still need some manual edits in some cases.

Reply:
thank you for your suggestion, I am looking into it

Reply:
I've kicked off the CI build. You might also want to import the codestyle into your IDE to avoid reformatting unrelated code - see https://github.com/airlift/codestyle#intellij-idea-2019-or-newer.

Reply:
fixed format, do you mind running the code 1 more time?